pkgs/by-name/ti/tigerbeetle/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, zig_0_11
+, testers
+, tigerbeetle
+, nix-update-script
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tigerbeetle";
+  version = "0.14.171";
+
+  src = fetchFromGitHub {
+    owner = "tigerbeetle";
+    repo = "tigerbeetle";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-MjsNQarRXsrWKJZ2aBi/Wc2HAYm3isLBNw81a75+nhc=";
+  };
+
+  nativeBuildInputs = [ zig_0_11.hook ];
+
+  zigBuildFlags = [
+    "-Dgit-commit=0000000000000000000000000000000000000000"
+    "-Dversion=${finalAttrs.version}"
+  ];
+
+  passthru = {
+    tests.version = testers.testVersion {
+      package = tigerbeetle;
+      command = "tigerbeetle version";
+    };
+    updateScript = nix-update-script { };
```

Comment:
I thought that the updateScript defaults to the nix-update-script. could be wrong thou.

Reply:
It doesn't, there's a "tip" box on the manual [here](https://nixos.org/manual/nixpkgs/unstable/#special-variables) (scroll down a bit) that shows the pattern of defining `updateScript` to be `nix-update-script {}`.

Reply:
Yep, they release a new version ~every week, which is why I set up an `updateScript` to automate at least a bit of the process. Once this PR gets merged I'll stay on top of the bot PRs to bump the version.